messaging/src/main/java/org/axonframework/queryhandling/SimpleQueryBus.java

```diff
@@ -229,17 +236,17 @@ private boolean unsubscribe(String queryName,
                 .thenAccept(monoSink::success)
                 .exceptionally(t -> {
                     logger.error(format("An error happened while trying to report an initial result. Query: %s", query),
-                                 t);
+                            t);
```

Comment:
From these changes, I am guessing you might not have set the style guide present in this project. Would you mind using the `axon_code_style.xml` file and set it as the style for this project?